resources/js/pages/profile/components/CurrentUserActivityList.tsx

```diff
@@ -0,0 +1,24 @@
+import { Flex } from "@chakra-ui/react";
+import Activity from "../../dashboard/components/ActivityCard";
+
+const CurrentUserActivityList = () => {
```

Comment:
Use generic name or general name
```suggestion
const UserActivityList = () => {
```

Reply:
Done